src/GRANULAR/fix_wall_gran.cpp

```diff
@@ -1261,6 +1261,7 @@ void FixWallGran::granular(double rsq, double dx, double dy, double dz,
 
   k_tangential = tangential_coeffs[0];
   damp_tangential = tangential_coeffs[1]*damp_normal_prefactor;
+  Fscrit = tangential_coeffs[2] * Fncrit;
```

Comment:
@dsbolin @jtclemm please note the changes in this commit. Our weekly static code analysis has reported that `Fscrit` was used before it was initialized.  You may want to check any simulations that used the affected features.

Reply:
Thanks for catching this. The bug was introduced by me very recently in PR #2698 , before that `Fscrit` was not used until after line 1347. Older simulations should be fine.